pkg/placement/dbsync.go

```diff
@@ -47,7 +47,6 @@ func DbSyncJob(
 	}
 
 	envVars := map[string]env.Setter{}
-	envVars["KOLLA_CONFIG_FILE"] = env.SetValue(KollaConfig)
```

Comment:
I think you can remove the global constant `KollaConfig` as well

Reply:
done